### ISSUE TITLE
Warning during automated GH builds 

### ISSUE THREAD

**Issue by shumonsharif** [quarkiverse/quarkus-cxf#231]
The automated GitHub builds show the following warning:
https://github.com/quarkiverse/quarkus-cxf/pull/226/checks?check_run_id=2640326484#step:5:176

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.quarkiverse.cxf.deployment.test.CxfClientTest
2021-05-21 14:48:37,098 WARN  [io.qua.boo.res.mav.BootstrapMavenContext] (main) Settings problem encountered at /usr/share/apache-maven-3.8.1/conf/settings.xml, line 164, column 16: org.apache.maven.settings.io.SettingsParseException: Unrecognised tag: 'blocked' (position: START_TAG seen ...</url>\r\n      <blocked>... @164:16) 
	at org.apache.maven.settings.io.DefaultSettingsReader.read(DefaultSettingsReader.java:87)
	at org.apache.maven.settings.building.DefaultSettingsBuilder.readSettings(DefaultSettingsBuilder.java:182)
	at org.apache.maven.settings.building.DefaultSettingsBuilder.build(DefaultSettingsBuilder.java:104)
	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.getEffectiveSettings(BootstrapMavenContext.java:252)
	at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.isOffline(BootstrapMavenContext.java:218)
```

**Comment by shumonsharif** [comment]
@gastaldi Is this an issue with the quarkiverse setup, and is there a need for concern?

**Comment by gastaldi** [comment]
@shumonsharif this is because Quarkus 1.13.4.Final is not compatible with Maven 3.8.1. This is fixed in https://github.com/quarkusio/quarkus/pull/16249 which is available only in the 2.0.0.Alpha builds. 

I marked to backport it.

**Comment by shumonsharif** [comment]
That works, thanks @gastaldi ... combing through the build logs, I see all the tests getting executed successfully.  I just wanted to verify that there weren't some unintended consequences due to the warning.  Closing this issue for now!
